src/help/help_impl.cpp

```diff
@@ -49,8 +50,10 @@
 
 #include <boost/algorithm/string.hpp>
 #include <cassert>                     // for assert
+#include <codecvt>
```

Comment:
Assuming `unicode_cast` works, we can drop this header and <locale> below.

Reply:
I don't know what you mean by "and below".

Reply:
Agh. That was supposed to say "and `<locale>` below." >_< There's an `#include <locale>` a few lines down.